rehostimage/chrome/locale/en-US/rehostimage.properties

```diff
@@ -35,6 +35,9 @@ error.downloadfailed=Could not re-download uncached image. Upload failed.
 error.resizefailed=Error resizing image. Upload failed.
 
 error.googleshortenfailed=Shortening URL with goo.gl failed. Using long URL.
+error.isgdshortenfailed=Shortening URL with is.gd failed. Using long URL.
+
+warning.pomf.shorteningnotsupported=URL shortening is not available for Pomf.
```

Comment:
Make it general "URL shortening is not available for this upload location." or add a {0} placeholder to insert the location name.